app/src/main/java/com/codesoom/assignment/product/ui/dto/ErrorResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.product.ui.dto;
+
+public class ErrorResponse {
+    private String message;
```

Comment:
여기에 `final`을 붙여줘도 괜찮겠습니다.
```suggestion
    final private String message;
```

Reply:
수정했습니다